gluten-substrait/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala

```diff
@@ -753,6 +764,8 @@ object ExpressionConverter extends SQLConfHelper with Logging {
           substraitExprName,
           expr.children.map(replaceWithExpressionTransformer0(_, attributeSeq, expressionsMap)),
           j)
+      case u: UnBase64 if SparkShimLoader.getSparkShims.unBase64FunctionFailsOnError(u) =>
```

Comment:
does this mean it has some limitation? 
Would better to add some notes here: 
https://github.com/apache/incubator-gluten/blob/main/docs/velox-backend-limitations.md#json-functions